src/components/Coins.css

```diff
@@ -0,0 +1,91 @@
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.coins-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(7rem, 1fr));
+}
+
+.coin-card {
+  display: flex;
```

Comment:
- [ ] Your card is missing some details as is the case of this arrow icon, please add that dear @mrcbq, as well as on your details page items

- [ ] TEMPLATE

<img width="257" alt="1" src="https://github.com/mrcbq/coinmarket/assets/77843050/f1dbe952-07d4-468e-aaff-e4c423338cbf">

- [ ] YOUR DESIGN

<img width="426" alt="2" src="https://github.com/mrcbq/coinmarket/assets/77843050/99499d02-fda1-4918-8786-28f8b32fe906">

Reply:
It's Done.